swagger/Platfrom API/bizaiChats.json

```diff
@@ -0,0 +1,407 @@
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "api_key": []
+          }
+        ],
+        "tags": [
+          "chatmng"
+        ],
+        "x-auth-type": "Directory"
+      }
+    },
+    "/v3/bizai-chats/{uid}": {
+      "get": {
+        "operationId": "ChatsController_getChat",
+        "summary": "Retrieve a chat",
+        "description": "Get a specific chat session based on its unique identifier (UID)",
+        "parameters": [
+          {
+            "name": "uid",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The unique identifier of the chat"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Chat retrieved",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "success": {
+                      "type": "boolean"
+                    },
+                    "data": {
+                      "$ref": "https://vcita.github.io/developers-hub/entities/bizaiChats.json"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "api_key": []
+          }
+        ],
+        "tags": [
+          "chatmng"
+        ],
+        "x-auth-type": "Directory"
+      }
+    },
+    "/v3/bizai-chats/{uid}/messages": {
+      "get": {
+        "operationId": "ChatsController_getMessages",
+        "summary": "List all messages",
+        "description": "Get all messages exchanged within a specific chat",
+        "parameters": [
+          {
+            "name": "uid",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The unique identifier of the chat"
+          },
+          {
+            "in": "query",
+            "name": "page",
+            "schema": {
+              "type": "integer",
+              "default": 1
+            },
+            "description": "Page number of the results."
+          },
+          {
+            "in": "query",
+            "name": "per_page",
+            "schema": {
+              "type": "integer",
+              "default": 25,
+              "maximum": 100
+            },
+            "description": "How many items to return per page."
+          },
+          {
+            "in": "query",
+            "name": "sort",
+            "schema": {
+              "type": "string",
+              "default": "updated_at:desc"
+            },
+            "description": "Sort by fields, e.g., \"created_at:asc,updated_at:desc\"."
+          },
+          {
```

Comment:
We are stopping the support for LHS filters, the filters should be explicitly defined.

Reply:
Do you want me to remove the filter query?

Reply:
yes, we only support explicit filters. things you defined and documented. There is a paragraph on this in the documentation